packages/tesseract.rb

```diff
@@ -0,0 +1,50 @@
+require 'package'
+
+class Tesseract < Package
+  description 'An OCR program'
+  homepage 'https://github.com/tesseract-ocr/tesseract'
+  @_ver = '4.1.1'
+  version @_ver
+  compatibility 'all'
+  source_url "https://github.com/tesseract-ocr/tesseract/archive/#{@_ver}.tar.gz"
+  source_sha256 '2a66ff0d8595bff8f04032165e6c936389b1e5727c3ce5a27b3e059d218db1cb'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/tesseract-4.1.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/tesseract-4.1.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/tesseract-4.1.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/tesseract-4.1.1-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '2784bfad438508abf959d5cf40c2cb7bad39ad30338cf94bc1b2cc2cdc4a4f5c',
+     armv7l: '2784bfad438508abf959d5cf40c2cb7bad39ad30338cf94bc1b2cc2cdc4a4f5c',
+       i686: '019d1cd38b1008e43e022b147a1b34ab11c11da1622733b7abea681215557ab3',
+     x86_64: '1661e2e92aeee9071cf37cee949a3c4d686fc379809e227be6e4b513b3a6b5f8'
+  })
+
+  depends_on 'libpng'
+  depends_on 'libtiff'
+  depends_on 'libjpeg_turbo'
```

Comment:
`libtiff` has `libjpeg` as a dependency.  Does this create a conflict?

Reply:
These packages have both as dependencies. This package doesn't change that and as such, the existence of this package is orthogonal to any conflict that might exist between `libtiff` and `libjpeg`

Reply:
See my earlier comment about the conflict betwee `libjpeg` and `libjpeg_turbo`.